config/routes.rb

```diff
@@ -8,6 +8,11 @@
 
   root to: "home#index"
 
+  resources :uploads do 
+    collection do
+      post :upload_zip
```

Comment:
I'm fine with this for now, we can refactor later, but we will definitely want to also support `rar` and `7z` files, I see those regularly, so I wonder if this is just better named "upload". I think actually that it could just be UploadController#create rather than a specifically-named action, so this could just be condensed down to: `resources :uploads, only: [:create]`